justfile

```diff
@@ -30,13 +30,13 @@ prqlc-lint:
 # Test prqlc
 packages := '--package=prqlc-ast --package=prqlc-parser --package=prql-compiler --package=prqlc'
 prqlc-test-fast:
-    cargo clippy --all-targets {{packages}} -- -D warnings
-
     # cargo insta test, but allowing multiple --package arguments
     INSTA_FORCE_PASS=1 cargo test --locked {{packages}}
```

Comment:
FYI this is now fixed upstream in `cargo-insta` so you can use it directly with multiple packages!

Reply:
Just in time for me to discover cargo-nextest :D

Reply:
Also supported by `cargo-insta`! :D